app/views/users/logix/edit.html.erb

```diff
@@ -1,46 +1,40 @@
-
-
 <div class="container">
   <div class="row feature">
 
     <div class="container">
 
       <h2>Profile</h2>
 
-      <!--<p> <strong>Name: </strong><%=@profile.name%></p>-->
-      <!--<p> <strong>Email: </strong><%=@profile.email%></p>-->
-      <!--<p> <strong>Member since: </strong><%=((Time.now.to_i - @profile.created_at.to_i)/1.day)%> days</p>-->
-      <!--<p> <strong>Educational Institution: </strong> IIIT-Bangalore </p>-->
-      <!--<p> <strong>Country: </strong> India </p>-->
-
-
-      <%= form_for @profile, html: { multipart: true},  url: {action: "update"}  do |f| %>
-          <div class="form-group">
-            <label for="name">Name:</label>
-            <input type="text" class="form-control" id="name" name="profile[name]" value = "<%= @profile.name%>">
-          </div>
-          <div class="form-group">
-            <label for="profile_picture">Profile Picture:</label>
-            <p> <%= image_tag @profile.profile_picture.url(:thumb)%></p>
-            <input type="file" class="form-control" id="profile_picture" name="profile[profile_picture]" value = "<%= @profile.profile_picture%>">
-          <!--<div class="form-group">-->
-            <!--<label for="educational_institution">Educational Institution:</label>-->
-            <!--<input type="text" class="form-control" id="educational_institution" placeholder="e.g. IIIT-Bangalore" name="profile[educational_institution]">-->
-          <!--</div>-->
-          <!--<div class="form-group">-->
-            <!--<label for="country">Country:</label>-->
-            <!--<input type="text" class="form-control" id="country" placeholder="Enter country" name="profile[country]">-->
-          <!--</div>-->
-          <button type="submit" class="btn btn-secondary">Save</button>
+      <%= form_for @profile, html: {multipart: true}, url: {action: "update"} do |f| %>
+        <div class="field form-group">
+          <%= f.label :name %><br>
+          <%= f.text_field :name, required: true, class:"form-control" %>
+        </div>
+
+        <div class="field form-group">
+          <%= f.label :country %><br/>
+          <%= f.country_select :country,priority_countries: ["IN"], class: "form-control" %>
+        </div>
+
+        <div class="field form-group">
+          <%= f.label :educational_institute %><br/>
+          <%= f.text_field :educational_institute,{ class: "form form-group form-control" } %>
```

Comment:
Here we should list list of educational institutions already available in our database as suggestions to avoid common typos. You can fetch them from `User.distinct.pluck(:educational_institute)`

Reply:
It should look like this http://twitter.github.io/typeahead.js/examples/

Reply:
so should i use typeahead.js in the project to achieve this?

Reply:
Not necessary, I just gave an example. You can use any other library that does this

Reply:
Ok, got it!

Reply:
So, I have added autocomplete to the educational institute option.I went with the typeahead.js library that you suggested.Please review the code and let me know if something needs to be improved.

![autocomplete](https://user-images.githubusercontent.com/20072704/53696119-f6df1480-3de9-11e9-974f-86316a70291d.gif)

Reply:
@tachyons , can you please check the changes i have made?

Reply:
Can you check if avatar is persisting, I see there was a custom code to handle it